controller/array_action/array_mediator_xiv.py

```diff
@@ -226,18 +258,22 @@ def copy_to_existing_volume_from_source(self, name, source_name, source_capacity
             logger.exception(ex)
             raise controller_errors.PermissionDeniedError("create vol : {0}".format(name))
 
-    def _get_vol_by_wwn(self, volume_id):
-        vol_by_wwn = self.client.cmd.vol_list(wwn=volume_id).as_single_element
-        if not vol_by_wwn:
+    def _get_cli_object_by_wwn(self, volume_id, not_exist_err=False):
```

Comment:
Why we change the method name from volume to object if the result is a volume?

Reply:
cause it can be a snapshot too